packages/fxa-auth-server/test/scripts/write-emails-to-disk.js

```diff
@@ -20,6 +20,7 @@ const execOptions = {
   env: {
     NODE_ENV: 'dev',
     LOG_LEVEL: 'error',
+    AUTH_FIRESTORE_EMULATOR_HOST: 'localhost:9090',
```

Comment:
I wonder why this is needed in all these test scripts -- I guess anywhere we're making use of Stripe objects needs to have authFirestore defined now in the auth sever?